Hunting Queries/OracleDatabaseAudit/OracleDBAuditUsersNewPrivilegesAdded.yaml

```diff
@@ -0,0 +1,31 @@
+id: 6be9634e-6661-4bff-98f1-4a743d7231a5
+name: OracleDBAudit - Users with new privileges
+description: |
+  'Query for searching user accounts whith new privileges.'
+requiredDataConnectors:
+  - connectorId: OracleDatabaseAudit
+    dataTypes:
+      - Syslog
+tactics:
+  - InitialAccess
+  - PrivilegeEscalation
+query: |
+  let lbtime_60d = 60d;
+  let lbtime_1d = 1d;
+  | where TimeGenerated between (ago(lbtime_60d) .. ago(lbtime_24h))
+  | where isnotempty(Privilege)
+  | where isnotempty(DstUserName)
+  | summarize Privileges = makeset(Privilege) by DstUserName;
+  | join (OracleDatabaseAuditEvent
+        | where TimeGenerated > ago(lbtime_24h)
+        | where isnotempty(DstUserName)
+        | where isnotempty(Privilege)
+        | project DstUserName, Privilege
+        ) on DstUserName
+  | where Privileges !contains Privilege

```

Comment:
Use `| where Privilege !in (Privileges)`

Reply:
updated

Reply:
looks like this did not get updated